source/_includes/dashboard/edit_dashboard.md

```diff
@@ -0,0 +1,7 @@
+{% capture name %}{{ include.name | default: page.title }}{% endcapture %}
+{% capture domain %}{{ include.domain | default: page.ha_domain }}{% endcapture %}
+{% capture title %}{{ include.title | default: page.title }}{% endcapture %}
+
+To add the {{ title | downcase }} to your user interface:
+1. In the top right of the screen, select the three dots menu and then select **Edit dashboard**. 
+2. In the bottom right corner, select the **Add card** button  and select from the card picker.
```

Comment:
```suggestion
2. In the bottom right corner, select the **Add card** button  and select from the card picker.

```
Let's add a new line here so text added on the line below where this is included is actually on the line below and not part of step 2.
Example for the conditional card <https://deploy-preview-29890--home-assistant-docs.netlify.app/dashboards/conditional/>